tampermonkey/PPT_Returns_AutoComplete.js

```diff
@@ -292,6 +319,42 @@ const confirmationPage = () => {
      }
 }
 
+/* ####################### Amend submitted returns pages */
+
+const submittedReturns = () => {
+     if (currentPageIs('/plastic-packaging-tax/submitted-returns')) {
+         document.getElementsByClassName('govuk-body')[1].click()
+     }
+}
+
+const existingReturn = () => {
+     if (currentPageIs('/plastic-packaging-tax/viewReturnSummary/22C2')) {
+         document.getElementById('amend-return').click()
+     }
+}
+
+const amendReturn = () => {
+     if (currentPageIs('/plastic-packaging-tax/amend-return')) {
+         let noChange = document.querySelector('#main-content > div > div > table:nth-child(2) > tbody > tr:nth-child(1) > td:nth-child(3) > span').innerText
+         if (noChange == 'Not yet amended') {
+             document.getElementById('amend-manufacture-link').click()
+         }
+     }
+}
+
+const amendManufacturedWeight = () => {
+     if (currentPageIs('/plastic-packaging-tax/amend-manufactured-weight')) {
+         document.getElementById('value').value = '251'
+         document.getElementsByClassName('govuk-button')[0].click()
+     }
+}
+
+const returnAmended = () => {
+     if (currentPageIs('/plastic-packaging-tax/return-amended')) {
+         document.querySelector('#main-content > div > div > p:nth-child(5) > a').click()
```

Comment:
Could this be updated to locate the element using the link text instead of the css?

Reply:
find by link text is a selenium specific command, the link it's clicking doesn't have identifiers on it at the moment, so css location was the backup.